cmd/utils/flags.go

```diff
@@ -1406,9 +1406,6 @@ func SetQuaiConfig(stack *node.Node, cfg *quaiconfig.Config, slicesRunning []com
 			cfg.Miner.GasPrice = big.NewInt(1)
 		}
 	}
-	if viper.GetString(EnvironmentFlag.Name) != params.LocalName {
-		cfg.Genesis.Nonce = viper.GetUint64(GenesisNonceFlag.Name)
```

Comment:
do we not need the nonce as a flag then ?

Reply:
fair, let me remove it

Reply:
fixed